Assets/Scripts/Player/PlayerStats.cs

```diff
@@ -0,0 +1,130 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using DG.Tweening;
+
+public class PlayerStats : MonoBehaviour
```

Comment:
For now I'd say keep this and we can pair it out to incorporate other things

Reply:
we're trying to stick to a code structure we've outlined in our UML diagram linked in the README. we'll probably need a big refactor to incorporate the behavior in this script according to our structure.